tensorboard/webapp/notification_center/_redux/notification_center_selectors_test.ts

```diff
@@ -57,5 +58,15 @@ describe('notification_center_selectors', () => {
         },
       ]);
     });
+
+    it('get last read timestamp', () => {
```

Comment:
can it be a `null`? If so, write a simple test.

Reply:
added.